<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_2">
          <property name="plainText">
           <string>This demo program shows a feature which is inconsistent between visual and braille display output.

Using NVDA 2019.1.1
And QT 5.10.1

Select 'Page' Tab, then Test Menu 1.
Focus moves to Tab 2 and plaintextedit text is displayed on the screen and Braille display.

Select 'Page' Tab, then Test Menu 2.
Focus moves to Tab 2, and the text of the plaintextedit is spoken, but the braille display shows the second entry in the list (on Tab 1), rather than the text of the plaintextedit.

Now select 'Page' Tab, then Test Menu 2 again.  This time, Focus moves to Tab 2, and the braille display correctly shows the text in the plaintextedit.
</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Test">
    <property name="title">
     <string>&amp;Test</string>
    </property>
    <addaction name="actionTest_1"/>
    <addaction name="actionTest_2"/>
    <addaction name="actionTest_3"/>
   </widget>
   <addaction name="menu_Test"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTest_1">
   <property name="text">
    <string>&amp;1 - Switch to PlainTextEdit</string>
   </property>
  </action>
  <action name="actionTest_2">
   <property name="text">
    <string>&amp;2 - Switch to PlainTextEdit and Select Item2 in List</string>
   </property>
  </action>
  <action name="actionTest_3">
   <property name="text">
    <string>&amp;3 - Swtich to PlainTextEdit and Select Item3 in List</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
